lingui.config.ts

```diff
@@ -1,14 +1,51 @@
 /** @type {import('@lingui/conf').LinguiConfig} */
 module.exports = {
-  locales: ['en', 'es', 'ja', 'fr', 'pt', 'zh', 'ru', 'de', 'uk', 'sv', 'fi', 'nl', 'el', 'it', 'pl'],
+  locales: [
```

Comment:
NIT: It is good that this list is in order.

Reply:
Fixed